pkg/cmd/server/origin/master.go

```diff
@@ -140,9 +139,13 @@ func (c *MasterConfig) InstallProtectedAPI(container *restful.Container) []strin
 	buildEtcd := buildetcd.New(c.EtcdHelper)
 	deployEtcd := deployetcd.New(c.EtcdHelper)
 	routeEtcd := routeetcd.New(c.EtcdHelper)
-	oauthEtcd := oauthetcd.New(c.EtcdHelper)
 	authorizationEtcd := authorizationetcd.New(c.EtcdHelper)
 
+	accessTokenStorage := accesstokenetcd.NewREST(c.EtcdHelper)
```

Comment:
Move down into the map.  These are never reused and doing it like this scrolls the usage onto a screen separate from the declaration.